sjopinie/sjopinie_app/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+
+class User(models.Model):
+    login = models.CharField(max_length=200, primary_key=True)
+    hash_of_password = models.CharField(max_length=255, blank=True)
+    date_of_registration = models.DateTimeField()
+
+
+class Lecturer(models.Model):
+    name = models.CharField(max_length=50)
+    surname = models.CharField(max_length=50)
+    academic_title = models.CharField(max_length=20)
+
+
+class Subject(models.Model):
+    name = models.CharField(max_length=80)
+    shortcut = models.CharField(max_length=10)
+
+    def __str__(self):
+        return self.shortcut
+
+
+class Opinion(models.Model):
+    author = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+    opinion_text = models.CharField(max_length=5000)
+    semester = models.CharField(max_length=10, null=True)
```

Comment:
może zrobić jak niżej z VOTE_VALUES?

models.CharField(choices=SEMESTER) 
gdzie SEMESTER = (
    (I, "I")
    . . . 
)

Reply:
Czy aby na pewno powinno być
```python
SEMESTER_VALUES = ((I, "I"))
#zamiast
SEMESTER_VALUES = ((1, "I"))
```

Reply:
Jeśli zmiana z commita ci pasuje to nie zapomnij kliknąć `Resolve Conversation`